src/calendar/js/calendar.js

```diff
@@ -123,6 +139,7 @@ Y.Calendar = Y.extend(Calendar, Y.CalendarBase, {
     _unhighlightCurrentDateNode : function () {
         var allHilitedNodes = this.get("contentBox").all("." + CAL_DAY_HILITED);
         if (allHilitedNodes) {
+            this._setTabIndex(allHilitedNodes, -1);
```

Comment:
allHilitedNodes is an array of nodes. Is _setTabIndex able to handle an array of nodes?

Reply:
You're right, setAttribute doesn't deal with lists. I will fix it to:

```
  for (let i = 0; i < allHilitedNodes.length; i++) {
      this._setTabIndex(allHilitedNodes[i], -1);
  }
```

But I couldn't find cases where more than one cell has the `CAL_DAY_HILITED` class at the same time. But I think we should do the .all() like this, there must be some reason.